bullet/src/KinematicsFeatures.cc

```diff
@@ -0,0 +1,98 @@
+#include <ignition/common/Console.hh>
+#include "KinematicsFeatures.hh"
+
+namespace ignition {
+namespace physics {
+namespace bullet {
+
+/////////////////////////////////////////////////
+FrameData3d KinematicsFeatures::FrameDataRelativeToWorld(
+    const FrameID &_id) const
+{
+  FrameData3d data;
+
+  // The feature system should never send us the world ID.
+  if (_id.IsWorld())
+  {
+    ignerr << "Given a FrameID belonging to the world. This should not be "
+           << "possible! Please report this bug!\n";
+    assert(false);
+    return data;
+  }
+
+  const auto linkID = _id.ID();
+  const auto &linkInfo = this->links.at(linkID);
```

Comment:
If we will the _id, which is a FrameID as a replacement to query the linkIDs I think we should something to exit the function cleanly in the case this function receives a not valid linkID.

I won't block this PR on that, but you can add a comment about it here so we remember later.

Reply:
Added on 25225aa